azure-pipelines.yml

```diff
@@ -7,6 +7,9 @@ steps:
   env:
     AZURE_PIPELINES: "true"
 
+- script: export NODE_OPTIONS="--max_old_space_size=4096"
```

Comment:
Don't forget to remove this code. It's already present in other section of the file.